config/Dockerfiles/rsync/rsync.sh

```diff
@@ -6,6 +6,10 @@ set -x
 #rsync_from="fedora-atomic@artifacts.ci.centos.org::fedora-atomic/f26/"
 #rsync_to="/home/output/"
 
+if [[ ! -v rsync_opts ]]; then
```

Comment:
Actually, that doesn't work if rsync_opts="" which is defined but empty, which is how I plan to use the variable when I don't want --delete